packages/stripe-graphql/src/schema/payment-methods.ts

```diff
@@ -0,0 +1,64 @@
+import { builder } from '../builder'
+
+builder.objectType('StripePaymentMethods', {
+  fields: (t) => ({
+    object: t.exposeString('object'),
+    url: t.exposeString('url'),
+    hasMore: t.exposeBoolean('has_more'),
+    data: t.expose('data', {
+      type: ['StripePaymentMethod'],
+      nullable: false
+    })
+  })
+})
+
+// TODO: generic args for payment methods
+// export const paymentMethodsArgs = builder.args((t) => ({
+//   type: t({ // unable to get this to work
+//     type: PaymentMethodTypes,
+//     required: true,
+//     defaultValue: 'card'
+//   }),
+//   startingAfter: t.string({
+//     required: false
+//   }),
+//   endingBefore: t.string({
+//     required: false
+//   }),
+//   limit: t.int({
+//     required: false
+//   })
+// }))
+
+export const PaymentMethodTypes = builder.enumType('PaymentMethodTypes', {
+  values: [
+    'acss_debit',
```

Comment:
It seems like the unofficial standard for graphql enums is to use all upper case values for enums.

This could be implemented as:

```ts
values: {
  ACSS_DEBIT: {  value:  'acss_debit' },
  AFFIRM: { value: 'affirm' }
  ...
} as const
```

I'm not too familiar with stripe, or if these values might be sourced from some other API/source/endpoint and need to match up (in that case, this conversion would not be ideal)